lib/src/dbus_remote_object_manager.dart

```diff
@@ -53,7 +53,10 @@ class DBusRemoteObjectManager extends DBusRemoteObject {
       {required String name, required DBusObjectPath path})
       : super(client, name: name, path: path) {
     var rawSignals =
-        DBusSignalStream(client, sender: name, pathNamespace: path);
+        DBusSignalStream(client,
+         sender: name,
+         /// Workaround for https://github.com/bus1/dbus-broker/issues/309
+         pathNamespace: path.value != '/' ? path : null);
```

Comment:
This is a little hard to read, I think it would be clearer if you set pathNamespace before creating the signal stream and with the more descriptive comment from https://gitlab.gnome.org/GNOME/glib/-/commit/3b28df1e008101341504f82c8e65f3109aca10cc, i.e.
```dart
String? pathNamespace = path.value;
//  Only add path_namespace if it's non-'/'. This removes a no-op key from
// the match rule, and also works around a D-Bus bug where
// path_namespace='/' matches nothing in D-Bus versions < 1.6.18.
if (pathNamespace == '/') {
   pathNamespace = null;
}